nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/PutHBaseRecord.java

```diff
@@ -127,6 +127,18 @@
             .defaultValue("1000")
             .build();
 
+    protected static final AllowableValue NULL_FIELD_EMPTY = new AllowableValue("empty-string", "Empty String", "Use an empty string");
```

Comment:
Shouldn't the name of this option be 'Empty Bytes'? As there's Record data types other than String, 'Empty Bytes' sounds more appropriate to me.

Reply:
I assume the purpose of putting an empty bytes is to empty an existing cell data in the column family. If so, I think we should mention that in the description of this and 'Skip Field' options so that user can understand ramifications, such as 'Skipping null value doesn't clear existing cell value.' .. etc

Reply:
My original intent was to make it easier on people writing Spark or MR jobs against HBase so they could assume in this case that the cell is there but has something akin to a null value. However, the points you made here do stand and should be added.